Core/src/main/java/com/plotsquared/core/command/Area.java

```diff
@@ -101,6 +104,8 @@
     private final WorldUtil worldUtil;
     private final RegionManager regionManager;
 
+    private final Map<UUID, Map<String, Object>> metaData = new HashMap<>();
```

Comment:
Because it made everything incredibly ugly and way too verbose. A player can't execute the same command more than once at any given moment, so it doesn't need to go through all of the meta protection stuff, as it's contained

Reply:
Fair. As long as it's only those few lines, I guess it's not worth to add a wrapper to get the casts away from this class